builds/otel.sh

```diff
@@ -6,7 +6,7 @@ DESTDIR="${DESTDIR}${otel_dir}"
 cd submodules/opentelemetry-java-contrib
 mkdir -p "$DESTDIR"
 ./gradlew --no-daemon :jmx-metrics:build
-cp jmx-metrics/build/libs/opentelemetry-jmx-metrics-*-SNAPSHOT.jar "$DESTDIR/opentelemetry-java-contrib-jmx-metrics.jar"
```

Comment:
Do you need to update `Dockerfile.windows` with the same change?

Reply:
Ah, I missed that one, yes. Pushed up the fix.